src/lib/holocene/input/chip-input.stories.svelte

```diff
@@ -100,3 +102,14 @@
     await userEvent.click(input);
   }}
 />
+
+<Story
+  name="Scroll Input Into View"
+  args={{ class: 'max-h-20 w-96', scrollTo: true }}
```

Comment:
> I think https://github.com/temporalio/ui/pull/2689 might make this unnecessary,

We can still set the max height on the component via the class. Might be nice to have this option if it comes up in the future.